src/main/webapp/brazilSearchResults.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
```

Comment:
**zxx** doesn't seem to be valid language code according to [this](https://www.w3schools.com/tags/att_global_lang.asp)

This shows up in the other files too

Reply:
I'll fix in a later pr